README.md

```diff
@@ -1,2 +1,49 @@
-# AllRides
+# All Rides
+
 Website for rent, sell and buy vehicles
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Overview
+
+Provide an overview of your project, explaining its purpose and key features. Give a brief introduction to Flask and SQLAlchemy and how they are used in your project.
+
+## Features
+
+List the main features of your project. For example:
+
+- User authentication and authorization
+- CRUD operations for managing data
+- Integration with a database using SQLAlchemy
+- RESTful API endpoints
+- Front-end templates using Jinja2
+
+## Installation
+
+Include instructions on how to install and set up your project locally. For example:
+
+1. Clone the repository: `git clone https://github.com/your-username/your-project.git`
```

Comment:
please add the proper git clone link

Reply:
Issue resolved, added correct git repository